demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -644,6 +644,41 @@ def test_validate_pack_readme_invalid_images(self):
         # this path is not an image path and should not be shown.
         assert 'https://github.com/demisto/content/raw/test3.png' not in errors
 
+    def test_validate_pack_readme_relative_url(self):
+        """
+            Given
+                - A pack README file with invalid relative path in it.
+            When
+                - Run validate on pack README file
+            Then
+                - Ensure:
+                    - Validation fails
+                    - Invalid relative paths were caught correctly
+                    - Invalid absolute paths were not caught.
+                    - Image paths were not caught
+        """
+        self.validator = PackUniqueFilesValidator(os.path.join(self.FILES_PATH, 'DummyPack2'))
```

Comment:
please use `Path` from pathlib 🙏

Reply:
As discussed over slack we'll leave as is :)

Reply:
what was the discussion here?